frontend/ui/widget/filechooser.lua

```diff
@@ -340,7 +344,9 @@ function FileChooser:getMenuItemMandatory(item, collate)
     local text
     if collate then -- file
         -- display the sorting parameter in mandatory
-        if collate == "date" then
+        if collate == "access" then
+            text = os.date("%Y-%m-%d %H:%M", item.attr.access)
```

Comment:
Thank you. Can you use the `datetime` function to format the date.

Reply:
Sure. It bugs me just a very, very tiny amount that the formatted date has two spaces if the hour value is under 10, e.g.
`2023-07-11  4:00 PM`
instead of
`2023-07-11 4:00 PM`

But I don't know what else that might break if that changed. A quick scan looks like those `secondsTo[X]` functions are all for display purposes only, but I'm not 100% sure they're not used for comparison somewhere I didn't notice.

Reply:
That's very much on purpose for display/alignment reasons, IIRC, yeah ;).